src/Engine/ProtoCore/Lang/CallSite.cs

```diff
@@ -832,6 +842,10 @@ private void UpdateCallsiteExecutionState(Object callsiteData, RuntimeCore runti
             out List<FunctionEndPoint> resolvesFeps,
             out List<ReplicationInstruction> replicationInstructions)
         {
+            replicationInstructions = null;
+            resolvesFeps = null;
+            Boolean flag = false;
```

Comment:
what does the flag indicate? Can we name it based on that?

Reply:
Yes that would be better. I will name it something like "MatchFound"